example/.mage/magefile.go

```diff
@@ -38,8 +38,8 @@ func init() {
 
 func All() {
 	mg.Deps(
-		mg.F(mgconvco.ConvcoCheck, "origin/master..HEAD"),
-		mggolangcilint.GolangciLint,
+		mg.F(mgconvco.CheckCommand, "origin/master..HEAD"),
```

Comment:
this should be moved to a separate function?

Reply:
Yeah I haven’t touched this one yet